cluster/azure/subnet/outputs.tf

```diff
@@ -1,4 +1,4 @@
-output "subnet_id" {
-  description = "The id of the vNet"
-  value       = azurerm_subnet.subnet.id
+output "subnet_ids" {
+  description = "The id of the subnet"
+  value       = azurerm_subnet.subnet.*.id
```

Comment:
Is it outputting all subnet id's that are created?

Reply:
There isn't a need, as the subnet is created with the cluster.  Data source uses vnet name and subnet name.  What scenario are you thinking?

Reply:
Got it. I can't think of a scenario now, but curious whether it is writing out `n` number of subnets or the last/first subnet id.

Reply:
Also - it's outputting all the subnets created (it takes in lists, so may create multiple).